src/applications/disability-benefits/all-claims/pages/individualsInvolvedFollowUp.js

```diff
@@ -0,0 +1,147 @@
+import { ptsd781NameTitle } from '../content/ptsdClassification';
+import {
+  individualsDescription,
+  personDescriptionText,
+} from '../content/individualsInvolved';
+import IndividualsInvolvedCard from '../components/IndividualsInvolvedCard';
+import fullNameUI from '../../../../platform/forms/definitions/fullName';
+
+import currentOrPastDateUI from 'us-forms-system/lib/js/definitions/currentOrPastDate';
+
+export const uiSchema = index => ({
+  'ui:title': ptsd781NameTitle,
+  'ui:description': individualsDescription,
+  [`incident${index}`]: {
+    personInvolved: {
+      'ui:title': ' ',
+      'ui:options': {
+        itemName: 'Individual',
+        viewField: IndividualsInvolvedCard,
+      },
+      items: {
+        name: fullNameUI,
+        personDescription: {
+          'ui:title': personDescriptionText,
+          'ui:widget': 'textarea',
+        },
+        'view:serviceMember': {
+          'ui:title': 'Were they a Servicemember',
+          'ui:widget': 'yesNo',
+        },
+        rank: {
+          'ui:title': 'What was their rank at the time of the event?',
+          'ui:options': {
+            expandUnder: 'view:serviceMember',
+          },
+        },
+        unitAssigned: {
+          'ui:title':
+            'What unit were they assigned to at the time of the event?',
+          'ui:options': {
+            expandUnder: 'view:serviceMember',
+          },
+        },
+        injuryDeathDate: currentOrPastDateUI(
+          'Date they were injured or killed',
+        ),
+        injuryDeath: {
+          'ui:title': 'How were they injured or killed?',
+          'ui:widget': 'radio',
+          'ui:options': {
+            labels: {
+              killedInAction: 'Killed in action',
+              woundedInAction: 'Wounded in action',
+              killedInNonBattle: 'Killed non-battle',
+              injuredInNonBattle: 'Injured non-battle',
+              other: 'Other',
+            },
+          },
+        },
+        injuryDeathOther: {
+          'ui:title': ' ',
+          'ui:options': {
+            expandUnder: 'injuryDeath',
+            expandUnderCondition: 'Other',
+          },
+        },
+        'view:individualAddMsg': {
+          'ui:title': ' ',
+          'ui:description':
+            'If anyone else was killed or injured in this event, you can add them now.',
+        },
+      },
+    },
+  },
+});
+
+export const schema = index => ({
+  type: 'object',
+  properties: {
+    [`incident${index}`]: {
+      type: 'object',
+      properties: {
+        personInvolved: {
+          type: 'array',
+          items: {
+            type: 'object',
+            properties: {
+              name: {
```

Comment:
Added this schema hardcoded, the updated `fullName` definition in `vets-json-schema` has suffix(that I don't need) and has first and last required(don't need). We'll be updated our 781 schema inside `vets-json-schema` and then porting those properties into the pages @dmccurdybah is currently working on that.